src/api/config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,2 @@
+# adding the number spaced by underscores to make rubocop happy
+StrongMigrations.start_after = 20_200_318_123_203
```

Comment:
I would prefer to just disable the cop here. It isn't a "large" number but a timestamp